assets/js/nav.ts

```diff
@@ -0,0 +1,23 @@
+export const enableNavAccordionLinks = (): void => {
```

Comment:
This was to handle the fact that otherwise the accordion links don't work. Perhaps there's a better pattern to accomplish that :man_shrugging: .